src/etc/cpu-usage-over-time-plot.sh

```diff
@@ -14,13 +14,19 @@
 #
 # Improvements to this script are greatly appreciated!
 
+if [[ $# != 2 ]]; then
+	echo "expected 2 arguments, recieved $#"
+	echo "example usage: './src/etc/cpu-usage-over-time-plot.sh 0003d1a38d97b12a389bd1e5b49f9db92094ee4a x86_64-linux-alt'"
+	exit 1
+fi
+
 set -ex
 
 bucket=rust-lang-ci2
 commit=$1
 builder=$2
 
-curl -O https://$bucket.s3.amazonaws.com/rustc-builds/$commit/cpu-$builder.csv
+curl -O https://$bucket.s3.amazonaws.com/rustc-builds-alt/$commit/cpu-dist-$builder.csv
```

Comment:
This probably shouldn't be changing to -alt, those are less typically queried than regular builders.